cmd/server/api/universalmintingrpchandler.go

```diff
@@ -15,32 +32,235 @@ type JSONRPCErrorResponse struct {
 	} `json:"error"`
 }
 
-// sendJSONRPCError sends a JSON RPC error response with the given message and code.
-func sendJSONRPCError(w http.ResponseWriter, code int, message string) {
+func (h *GlobalRPCHandler) UniversalMintingRPCHandler(w http.ResponseWriter, r *http.Request) {
+	josonRPCRequest := h.GetJsonRPCRequest()
+	var params ParamsRPCRequest
+	if len(josonRPCRequest.Params) == 0 || json.Unmarshal(josonRPCRequest.Params[0], &params) != nil {
+		http.Error(w, "Error parsing params or missing params", http.StatusBadRequest)
+		return
+	}
+
+	var blockNumber string
+	if len(josonRPCRequest.Params) == 2 {
+		if err := json.Unmarshal(josonRPCRequest.Params[1], &blockNumber); err != nil {
+			http.Error(w, "Error parsing block number", http.StatusBadRequest)
+			return
+		}
+	}
+	slog.Info("block number", "blockNumber", blockNumber)
```

Comment:
What occurs if the length of `josonRPCRequest.Params != 2`, we log an empty string as "blockNumber"?